var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -354,6 +354,8 @@ def hostconfig(self, spec, prefix):
         if cxxflags:
             cfg.write(cmake_cache_entry("CMAKE_CXX_FLAGS", cxxflags))
         fflags = ' '.join(spec.compiler_flags['fflags'])
+        if '%cce' in self.spec:
+            fflags += "-ef"
```

Comment:
IIRC this is not consistent with a change you made in a PR that has been merged.

Reply:
@tldahlgren Do you still want this addressed? I believe fixed this in a subsequent commit?